cpp/harvest/nacjd.cc

```diff
@@ -158,26 +165,52 @@ void ExtractExistingIDsFromMarc(MARC::Reader * const marc_reader, std::set<std::
     }
 }
 
+void ExtractOneIDFromWebsite(const unsigned nacjd_id) {
+    std::string nacjd_title_path("/tmp/nacjd_titles.html");
```

Comment:
It seems like this function was partially copy/pasted from `ExtractIDsFromWebsite`. In that other function, the HTML file has the purpose to get the list of all IDs from it.

However, in this new function we only want to extract a single dataset whereof the ID is already known via the parameter nacjd_id. So i suppose we do not need the HTML file in this function at all and it can be entirely removed.